tests/libcxx/usual_matrix.lst

```diff
@@ -3,7 +3,8 @@
 
 RUNALL_INCLUDE ..\universal_prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/EHsc /MTd /std:c++latest /permissive- /FImsvc_stdlib_force_include.h /wd4643"
+# PM_CL="/EHsc /MTd /std:c++latest /permissive- /FImsvc_stdlib_force_include.h /wd4643"
+PM_CL="/kernel /std:c++latest /permissive- /FImsvc_stdlib_force_include.h /wd4643"
 RUNALL_CROSSLIST
 PM_CL="/analyze /Zc:preprocessor"
-PM_COMPILER="clang-cl" PM_CL="-fno-ms-compatibility -fno-delayed-template-parsing"
+# PM_COMPILER="clang-cl" PM_CL="-fno-ms-compatibility -fno-delayed-template-parsing"

```

Comment:
This file shouldn't be merged, as it is just a prototype hack.  We will need to figure out what to do with the /EHsc, /MTd, and /kernel flags at some point.